lua/cmake-tools/init.lua

```diff
@@ -1238,7 +1294,7 @@ function cmake.compile_commands_from_soft_link()
     return
   end
 
-  local source = vim.loop.cwd()
+  local source = config.working_dir
     .. "/"
     .. config.build_directory.filename
     .. "/compile_commands.json"
```

Comment:
Why should destination be `vim.loop.cwd() .. "/compile_commands.json"` instead of `config.working_dir .. "/compile_commands.json"`

Reply:
Make sense